lib/fingerprint.browser.js

```diff
@@ -0,0 +1,12 @@
+var pad = require('./pad.js');
+
+var env = typeof window === 'object' ? window : self;
```

Comment:
`self` is a reference to the global scope (equivalent to `window`) in WebWorkers and ServiceWorkers: https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/self

I looked at a couple of modules designed to work in both a browser context and in WebWorkers and this was the pattern they use.